content/posts/2019-12-28-a-look-back-at-2019.md

```diff
@@ -1,51 +1,51 @@
-After a few years of working on the web platform as a front end developer, this year threw me on the deep end of a server-side. I am currently working on a transition from monolithic, outdated codebase to modern, serverless architecture for a huge UK based client (sorry, cannot reveal a name). I am extremely happy to have an opportunity to embrace the web from the backend perspective but the first few months were not smooth. Being surrounded by engineers familiar with concepts that I have never heard of wasn't fun. My understanding of the new stack quickly improved. I managed to transition anxiety to an opportunity to learn. I am glad and confident to say that I am a much better programmer than I use to be twelve months ago.
+After a few years of working on web platforms as a front end developer, this year threw me into the deep end of server-side. I am currently working on a transition from monolithic, outdated codebase to modern, serverless architecture for a huge UK based client (sorry, cannot reveal a name). I am extremely happy to have an opportunity to embrace the web from the backend perspective but the first few months were not smooth. Being surrounded by engineers familiar with concepts that I have never heard of wasn't fun. My understanding of the new stack quickly improved. I managed to transition anxiety to an opportunity to learn. I am glad and confident to say that I am a much better programmer than I use to be twelve months ago.
 
-For some people, professional life is everything between 9:00 AM and 5:00 PM. That's not the case for me. I manage to accomplish the best piece of productive work outside of these boundaries. The website that you are reading now has been created outside of my day-to-day job and became one of my biggest achievements ever. It exploded this year and it is bigger than ever before counting around 8 thousands unique visitors every single day. I published around 30 articles this year, received hundreds of comments and some of my articles has been reposted on multiple reputable websites. Overall it was another fantastic year for me being a contributor to the open web platform and I would like to thank you all for reading, feedback and support.
+For some people, development is something that only happens between 9:00 AM and 5:00 PM. That's not the case for me. I manage to accomplish some of the best pieces of productive work outside of these boundaries. The website that you are reading now has been created outside of my day-to-day job and has become one of my biggest achievements ever. It exploded this year and it is bigger than ever before, counting around 8 thousands unique visitors every single day. I published around 30 articles this year, received hundreds of comments and some of my articles has been reposted on multiple reputable websites. Overall it was another fantastic year for me being a contributor to the open web platform and I would like to thank you all for reading, your feedback and your support.
```

Comment:
@pawelgrzybek  I wasn't quite sure what you were saying here, do you work outside of these hours and do your best work then? Or do you think people only think of development during work, and some of your best code is non-work related?